cmd/kubeadm/app/master/apiclient.go

```diff
@@ -73,11 +73,10 @@ func CreateClientAndWaitForAPI(file string) (*clientset.Clientset, error) {
 		cs, err := client.ComponentStatuses().List(v1.ListOptions{})
 		if err != nil {
 			if apierrs.IsForbidden(err) {
-				fmt.Print("\r[apiclient] Waiting for the API server to create RBAC policies")
+				fmt.Println("[apiclient] Waiting for API server authorization")
```

Comment:
if the client appears to hang as long as this is true, printing something while it's breaking on this state seems good to me (see the printlns in the other shortcircuiting checks in this Poll)